AuthorizationTest

MaryManzhos opened the pull request.

MaryManzhos committed:
Add AuthorizationTest

MaryManzhos committed:
Add DashboardPage

dzmitryrak commented on src/test/java/data/TestData.java:
```diff
@@ -0,0 +1,7 @@
+package data;
+
+public class TestData {
+    public static final String URL = "https://marymanzhos.testrail.io/";
```
Прятать через PropertyManager и переменные среды

dzmitryrak commented on src/test/java/tests/BaseTest.java:
```diff
@@ -0,0 +1,29 @@
+    @BeforeClass
+    public void setUp() {
+
+        if (System.getProperty("os.name").contains("Windows")){
```
давай через нормальный CapabiltiesGenerator

MaryManzhos committed:
Added CapabilitiesGenerator
added ProjectTest

MaryManzhos committed:
Added PropertyReader

MaryManzhos committed:
added allure reporting
added RetryAnalyzer
added TestListener
added create project with API
added Tests: Create, Delete, Update project
